PWGHF/HFC/Tasks/taskCorrelationDplusHadrons.cxx

```diff
@@ -313,6 +314,7 @@ struct HfTaskCorrelationDplusHadrons {
         }
       }
       registry.fill(HIST("hMassDplusVsPt"), massD, ptD, efficiencyWeightD);
+      registry.fill(HIST("hMassDplusVsPtWoeff"), massD, ptD, 1.0);
```

Comment:
Since for the histogram no _weight_ is needed, I suggest removing the 1.0
Same for line 415 and 425.
```suggestion
      registry.fill(HIST("hMassDplusVsPtWoEff"), massD, ptD);
```